### DISCUSSION TITLE
Difference between "Quick access" & "Recent files"

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#2831] [Q&A] [answered]
What is the difference between Quick access & Recent files (from Amaze sidebar) ?

Currently Quick access shows recent files in reversed order...& a single file 4-5 times in the in the list (same goes for all files in the QA list)

QA loading for a long time & sometimes crashes #2722

---
[FR]

![20210922_184720](https://user-images.githubusercontent.com/48226312/134350749-eb387fc8-3b8e-48f3-97dc-58f846954ee0.jpg)

mention a short description of what it does

**Comment by EmmanuelMess** [comment]
@VishalNehra

**Comment by VishnuSanal** [comment, marked as answer]
The only difference will be that one's from the system & other is Amaze's history (we've history dialog that shows the same thing too). We should discuss over this & choose whether even we should have both these options in the coming releases.

**Reply by VishnuSanal** [reply]
> We should discuss over this & choose whether even we should have both these options in the coming releases.

@VishalNehra @EmmanuelMess @TranceLove

**Reply by EmmanuelMess** [reply]
I would say to throw out quick accesses. And keep the rest as is.

**Reply by VishalNehra** [reply]
As explained by @VishnuSanal quick access are the files recently opened using amaze, but recent files are ones which were recently modified by the system. We shouldn't remove either as both have different usecase. 
I can notice there's a crash using quick access when we add a cloud storage, we should fix it and we'll be good.
See #3048

**Reply by EmmanuelMess** [reply]
> quick access are the files recently opened using amaze, but recent files are ones which were recently modified by the system. We shouldn't remove either as both have different usecase.

How should the user know that? There is at least a naming issue here.

Also, this presupposes the user assumes that "the system" and "an app in the system" are different things, maybe the user doesn't presuppose that.

**Reply by freddyhayward** [reply]
"Recent files" should be renamed to "Recently modified".

"Quick access" should be renamed to something else, though I'm not sure what. "Recently opened", "Recently accessed", "History", or "Recently opened in Amaze" would all be better.

If both of these are renamed, then https://github.com/TeamAmaze/AmazeFileManager/issues/2896 would no longer be necessary.

**Reply by shuvashish76** [reply]
![amaze](https://user-images.githubusercontent.com/48226312/150351429-2000634f-74be-477d-99f4-e7c445ddf54d.png)

#3134
